lib/DoctrineCodingStandard/Sniffs/Classes/ExceptionInterfaceNamingSniff.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineCodingStandard\Sniffs\Classes;
+
+use DoctrineCodingStandard\Helpers\UseStatementHelper;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use const T_EXTENDS;
+use const T_INTERFACE;
+use const T_OPEN_CURLY_BRACKET;
+use function array_combine;
+use function array_map;
+use function count;
+use function explode;
+use function preg_grep;
+use function preg_match;
+use function trim;
+
+final class ExceptionInterfaceNamingSniff implements Sniff
+{
+    private const CODE_NOT_AN_EXCEPTION = 'NotAnException';
+
+    /**
+     * {@inheritdoc}
```

Comment:
Drop, or rather document properly as `@return mixed[]`.

Reply:
This would be `@return int[]` for this method. All `T_*` constants are integers.

Reply:
Not all, some are strings too. (Can't remember which exactly.)

Reply:
I didn't found ones that are strings. https://3v4l.org/aCWko
I'll keep `int[]`.

Reply:
Constants defined by PHPCS itself are strings: https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Util/Tokens.php#L12

Token constants in PHP are only integers.

Reply:
Yep, those. :)

Reply:
I added `string[]` for the PHPCS ones to the return type.